tests/unit/responses/aps/PhysRevX.7.021022_parsed.yml

```diff
@@ -24,3 +24,702 @@ license_url: https://creativecommons.org/licenses/by/4.0/
+    dois:
+    - 10.1103/PhysRevLett.96.181602
+    label: '15'
+- reference:
+    publication_info:
+      journal_volume: '42'
+      page_start: '2323'
+      year: 2010
+      artid: '2323'
+      journal_title: Gen. Relativ. Gravit.
+    authors:
+    - inspire_role: author
+      full_name: Van Raamsdonk, M.
+    title:
+      title: Building up Spacetime with Quantum Entanglement
+    dois:
+    - 10.1007/s10714-010-1034-0
+    label: '16'
+- reference:
+    publication_info:
+      journal_volume: '2007'
+      page_start: '062'
+      journal_issue: '07'
+      artid: '062'
+      journal_title: J. High Energy Phys.
+    authors:
+    - inspire_role: author
+      full_name: Hubeny, V.E.
+    - inspire_role: author
+      full_name: Rangamani, M.
+    - inspire_role: author
+      full_name: Takayanagi, T.
+    title:
+      title: A Covariant Holographic Entanglement Entropy Proposal
+    dois:
+    - 10.1088/1126-6708/2007/07/062
+    label: '17'
+- reference:
+    publication_info:
+      journal_volume: '29'
+      year: 2012
+      artid: '155009'
+      journal_title: Classical Quantum Gravity
+    authors:
+    - inspire_role: author
+      full_name: Czech, B.
+    - inspire_role: author
+      full_name: Karczmarek, J.L.
+    - inspire_role: author
+      full_name: Nogueira, F.
+    - inspire_role: author
+      full_name: Van Raamsdonk, M.
+    title:
+      title: The Gravity Dual of a Density Matrix
+    dois:
+    - 10.1088/0264-9381/29/15/155009
+    label: '18'
+- reference:
+    publication_info:
+      journal_volume: '2016'
+      page_start: 068
+      journal_issue: 09
+      artid: 068
+      journal_title: J. High Energy Phys.
+    authors:
+    - inspire_role: author
+      full_name: Jafferis, D.L.
+    - inspire_role: author
+      full_name: Suh, S.J.
+    title:
+      title: The Gravity Duals of Modular Hamiltonians
+    dois:
+    - 10.1007/JHEP09(2016)068
+    label: '19'
+- reference:
+    publication_info:
+      journal_volume: '31'
+      year: 2014
+      artid: '225007'
+      journal_title: Classical Quantum Gravity
+    authors:
+    - inspire_role: author
+      full_name: Wall, A.
+    title:
+      title: Maximin Surfaces, and the Strong Subadditivity of the Covariant Holographic
+        Entanglement Entropy
+    dois:
+    - 10.1088/0264-9381/31/22/225007
+    label: '20'
+- reference:
+    publication_info:
+      journal_volume: '117'
+      year: 2016
+      artid: '021601'
+      journal_title: Phys. Rev. Lett.
+    authors:
+    - inspire_role: author
+      full_name: Dong, X.
+    - inspire_role: author
+      full_name: Harlow, D.
+    - inspire_role: author
+      full_name: Wall, A.C.
+    title:
+      title: Reconstruction of Bulk Operators within the Entanglement Wedge in Gauge-Gravity
+        Duality
+    dois:
+    - 10.1103/PhysRevLett.117.021601
+    label: '21'
+- reference:
+    persistent_identifiers:
+    - value: arXiv:hep-th/1601.07616
```

Comment:
this is wrong, an arXiv identifier is not a handle. This is probably due to a weird mixup of old and new-style identifiers. Can you amend the builder to try harder to treat this as an arXiv eprint number? the result should be `arxiv_eprint: 1601.07616`.